Source/Utilities/StringContains.lua

```diff
@@ -0,0 +1,15 @@
+function Auctionator.Utilities.StringContains(s, sub, ...)
+  if s == nil or sub == nil or sub == "" then
+    return false;
```

Comment:
Remove `;`

Reply:
This is the function extracted from zc, so I left it "as is". I'll fix it anyway 😛